CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 3.1.1
+
+## Improvements
+
+A unique id has been added to the Portal component entrance and exit nodes.
```

Comment:
It'd be better if this explained why this is an improvement, or what problem adding the unique id fixes.